packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testMatch1To3Indentations.st

```diff
@@ -0,0 +1,5 @@
+tests - matching
+testMatch1To3Indentations
+
+	#(' #' '  #' '   #') do: [:string |
```

Comment:
I prefer that we would decide when to use `{}` and when `#()` for array literals.
I think, in cases when we really want an array of symbols, `#()` is appropiate:
https://github.com/hpi-swa-teaching/MarkdownEditor/blob/3448307b24dbe24d713d603e2018d25a12dcf931/packages/MarkdownEditor-Tests.package/MarkdownEditorTest.class/instance/expectedFailures.st#L5
In other cases, like this test, I would prefer `{}`.

Also applies to the other multi-case tests.

Reply:
I thought it is kind of a way to represent that these values remain constant. Why shouldn't `#('...')` be appropriate? The only thing that is kinda annoying is the fact that you cannot really see the separation between the strings.

Reply:
I like the `#()` for constants.

Reply:
Where would you draw the boundary between `#()` and `{}`?
We have arrays in many of our tests and were always using `{}`. In most cases, `#()` would also work, except when the `#()` would transform class names into symbols. E.g. it would break
https://github.com/hpi-swa-teaching/MarkdownEditor/blob/3448307b24dbe24d713d603e2018d25a12dcf931/packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/testBlockStructureHeadingWithParagraph.st#L5
Also, `#(1->10 2->8)` would transform to something like `#(1 #-> 10 2 #-> 8)`; they don't support dictionaries in the way `{}` does.

So my proposal stands to only use `#()` when the items are actually symbols. In other cases, `{}` just works as expected.
But if you come up with a simple rule which won't always raise discussions whether to write `{}` or `#()`, I'm also happy.